src/styles/index.scss

```diff
@@ -1,3 +1,126 @@
+@import url("https://fonts.googleapis.com/css2?family=Manrope:wght@400;700&family=Roboto:wght@400;500;700&display=swap");
+
+$main-accent: #060b35;
+$secondary: #616070;
+$blue-accent: #00acdc;
+
 body {
   margin: 0;
+  font-family: Roboto, Helvetica, sans-serif;
+}
+
+.card {
+  box-sizing: border-box;
```

Comment:
it s better to apply this property to all elements (use * selector)

when all elements have the same `box-sizing` value, their behavior becomes more predictable